plugins/ingest/src/main/java/org/elasticsearch/plugin/ingest/IngestModule.java

```diff
@@ -52,41 +32,12 @@ public IngestModule(boolean ingestEnabled) {
 
     @Override
     protected void configure() {
-        // Even if ingest isn't enable we still need to make sure that rest requests with pipeline
+        // Even if ingest isn't enabled we still need to make sure that rest requests with pipeline
         // param copy the pipeline into the context, so that in IngestDisabledActionFilter
         // index/bulk requests can be failed
         binder().bind(IngestRestFilter.class).asEagerSingleton();
```

Comment:
This really shouldn't require a module, but should be registered the same way a rest action is. I think we should have a follow up issue to remove registering anything against RestController directly (similar to the PR @nik9000 has open for rest actions).